### PR TITLE
Add maintainers minutes draft for Jul 2024

### PR DESCRIPTION
**Pull request by jfy133**
@netlify /blog/2024/maintainers-minutes-2024-07-01

### CODE REVIEW

File: src/content/blog/2024/maintainers-minutes-2024-07-01.mdx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+## Cleaning up our closet
+
+🧹 The second working group, named 'test-data task-force' was set up to clean up the nf-core test-data repository.
+
+In various areas of nf-core, it has been brought up that submitting and using the test-data repository is not optimal.
+Community members have said they find there turnaround time for reviews on the test-data repository.
+Furthermore, members have said that knowing where to put, and then where to find existing test-data is hard to find due to insufficient documentation.
+In particular, the `delete_me/` folder has ballooned turning into a wild-west, contributing to the difficulties finding data.
+
+⛏ This working group will be led by Simon, and if you want to give your input or get involved, you can join the [#wg-test-data-task-force](https://nfcore.slack.com/archives/C07B5FK9GKA) slack channel.
```
Comment (FriederikeHanssen) [inline src/content/blog/2024/maintainers-minutes-2024-07-01.mdx:57]: should we link people with their github profile?

File: src/content/blog/2024/maintainers-minutes-2024-07-01.mdx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+## Ensuring quality
+
+☁️ Finally, there was a discussion about AWS megatests runs (runs of full-sized 'realistic' data executed on each pipeline release) regularly failing.
+
+It was brough up this often happens due to conflicts with Seqera's [Fusion](https://seqera.io/fusion/) file system, but more critically - these were only being discovered _after_ release.
+
+It was decided that documentation will be improved and that pipeline developers will be made aware they are allowed to trigger (a limited number) of manual full-test run.
+
+🤖 Matthias would also investigate to see if we can semi-automate a test being triggered prior release (without them. executing too many times).
```
Comment (FriederikeHanssen) [inline src/content/blog/2024/maintainers-minutes-2024-07-01.mdx:67]:
```suggestion
🤖 Matthias would also investigate to see if we can semi-automate a test being triggered prior release (without them executing too many times).
```

**Comment by mashehu** [comment]
After you added Rike's suggestions, can you please close this PR and open it from a fork of the branch `monorepo` (the files need to then be instead be in `sites/main-site/src/content/blog`). sorry for the current limbo

File: src/content/blog/2024/maintainers-minutes-2024-07-01.mdx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+by providing brief summaries of the monthly Maintainer's team meetings.
+
+## End of the subworkflow naming saga?
+
+📚 The first point of order was reviewing our homework from last time.
+
+We reviewed various proposals put forward for how nf-core should name our subworkflows, and after comparison, it was decided to go with a less strict or structured name, and go with more flexible and descriptive names.
+This allows subworkflows to be understandable in what they do, without making them too long.
+In general short subworkflows with only one tool option in each step can simply list the main modules used, whereas longer subworkflows with more option can be descriptive without explicitly listing all modules.
+To compensate the loss of detail, Matthias will investigate adding more metadata and keywords to the [nf-core subworkflow](https://nf-co.re/subworkflows) page for findability purposes.
```
Comment (jfy133) [inline src/content/blog/2024/maintainers-minutes-2024-07-01.mdx:35]:
```suggestion
To compensate the loss of detail, Matthias (@mashehu) will investigate adding more metadata and keywords to the [nf-core subworkflow](https://nf-co.re/subworkflows) page for findability purposes.
```

File: src/content/blog/2024/maintainers-minutes-2024-07-01.mdx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+✍ Jon has been tasked formalising the discussions in proper guidelines, and will be posted on the nf-core website [guidelines page](https://nf-co.re/docs/guidelines/components/subworkflows) soon.
```
Comment (jfy133) [inline src/content/blog/2024/maintainers-minutes-2024-07-01.mdx:37]:
```suggestion
✍ Jon (@pinin4fjords) has been tasked formalising the discussions in proper guidelines, and will be posted on the nf-core website [guidelines page](https://nf-co.re/docs/guidelines/components/subworkflows) soon.
```

File: src/content/blog/2024/maintainers-minutes-2024-07-01.mdx [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+## Daisy chains
+
+🌻 We discussed a recent pipeline proposal from Maxime which acts as one of the first nf-core forays into 'daisy-chaining' pipelines (i.e., importing pipelines into other pipelines).
+
+In this case the maintainers team collectively decided against making it an official pipeline at the moment.
+This was primarily due to concerns about a current lack of tooling and infrastructure in this context, particularly in how would documentation be 'merged' together, but also due to unresolved 'philosophical' discussions on what is the distinction between a subworkflow versus a workflow.
+
+🤔 However given the burning need by the community to have such functionality, we decided to set up the first (of two!) 'working groups', let by Rike, to address this question. Feel free to join [#wg-meta-pipelines](https://nfcore.slack.com/archives/C07B5FK9GKA) join in.
```
Comment (jfy133) [inline src/content/blog/2024/maintainers-minutes-2024-07-01.mdx:46]:
```suggestion
🤔 However given the burning need by the community to have such functionality, we decided to set up the first (of two!) 'working groups', let by Rike (@FriederikeHanssen), to address this question. Feel free to join [#wg-meta-pipelines](https://nfcore.slack.com/archives/C07B5FK9GKA) join in.
```

File: src/content/blog/2024/maintainers-minutes-2024-07-01.mdx [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+## Cleaning up our closet
+
+🧹 The second working group, named 'test-data task-force' was set up to clean up the nf-core test-data repository.
+
+In various areas of nf-core, it has been brought up that submitting and using the test-data repository is not optimal.
+Community members have said they find the turnaround time for reviews on the test-data repository take too long.
+Furthermore, members have said that knowing where to put, and then where to find existing test-data is hard to find due to insufficient documentation.
+In particular, the `delete_me/` folder has ballooned turning into a wild-west, contributing to the difficulties finding data.
+
+⛏ This working group will be led by Simon, and if you want to give your input or get involved, you can join the [#wg-test-data-task-force](https://nfcore.slack.com/archives/C07B5FK9GKA) slack channel.
```
Comment (jfy133) [inline src/content/blog/2024/maintainers-minutes-2024-07-01.mdx:57]:
```suggestion
⛏ This working group will be led by Simon (@SPPearce), and if you want to give your input or get involved, you can join the [#wg-test-data-task-force](https://nfcore.slack.com/archives/C07B5FK9GKA) slack channel.
```

File: src/content/blog/2024/maintainers-minutes-2024-07-01.mdx [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+We reviewed various proposals put forward for how nf-core should name our subworkflows, and after comparison, it was decided to go with a less strict or structured name, and go with more flexible and descriptive names.
+This allows subworkflows to be understandable in what they do, without making them too long.
+In general short subworkflows with only one tool option in each step can simply list the main modules used, whereas longer subworkflows with more option can be descriptive without explicitly listing all modules.
+To compensate the loss of detail, Matthias will investigate adding more metadata and keywords to the [nf-core subworkflow](https://nf-co.re/subworkflows) page for findability purposes.
+
+✍ Jon has been tasked formalising the discussions in proper guidelines, and will be posted on the nf-core website [guidelines page](https://nf-co.re/docs/guidelines/components/subworkflows) soon.
+
+## Daisy chains
+
+🌻 We discussed a recent pipeline proposal from Maxime which acts as one of the first nf-core forays into 'daisy-chaining' pipelines (i.e., importing pipelines into other pipelines).
```
Comment (jfy133) [inline src/content/blog/2024/maintainers-minutes-2024-07-01.mdx:41]:
```suggestion
🌻 We discussed a recent pipeline proposal from Maxime (@maxulysse) which acts as one of the first nf-core forays into 'daisy-chaining' pipelines (i.e., importing pipelines into other pipelines).
```

File: src/content/blog/2024/maintainers-minutes-2024-07-01.mdx [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+
+## Ensuring quality
+
+☁️ Finally, there was a discussion about AWS megatests runs (runs of full-sized 'realistic' data executed on each pipeline release) regularly failing.
+
+It was brough up this often happens due to conflicts with Seqera's [Fusion](https://seqera.io/fusion/) file system, but more critically - these were only being discovered _after_ release.
+
+It was decided that documentation will be improved and that pipeline developers will be made aware they are allowed to trigger (a limited number) of manual full-test run.
+
+🤖 Matthias would also investigate to see if we can semi-automate a test being triggered prior release (without them executing too many times).
```
Comment (jfy133) [inline src/content/blog/2024/maintainers-minutes-2024-07-01.mdx:67]:
```suggestion
🤖 Matthias (@mashehu) would also investigate to see if we can semi-automate a test being triggered prior release (without them executing too many times).
```

**Pull request closed without merging**